### PR TITLE
feat: Document follow

### PR DESCRIPTION
**Pull request by Anurag810**

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Cases where the document will be __automatically__ followed after activating Document Follow:
+
+* When you create a new document.
+* When you modify a document.
+* When you like a document using the heart icon.
+* When another user shares a document with you or assigns you to a document, you'll be following the document.
+
+This is what an email notification for Document Follow will look like:
+
+<img class="screenshot" alt="Defining Criteria" src="{{docs_base_url}}/assets/img/setup/email/document-follow-email-looks-like.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md]: remove looks-like from the name

File: erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+1. Go to: **Settings > My Settings**.
+1. In the Document Follow section, tick the 'Send Notifications for documents followed by me' checkbox.
+1. Set the frequency at which you want to receive emails.
+1. Now, you can receive emails for the documents you follow.
+
+## 1.1 How to Follow documents
+1. Go to any document, you'll see an option to follow the document.
+1. Click on 'Follow'.
+
+<img class="screenshot" alt="Defining Criteria" src="{{docs_base_url}}/assets/img/setup/email/document-follow-how-to-follow.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md]: the red box color is darker than usual, use the default red color

File: erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<!-- add-breadcrumbs -->
+# Document Follow
+
+> Introduced in v12
+
+**Document Follow is feature that allows users to track changes in documents via email notifications.**
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md]: Document Follow is a feature that allows users to track changes in documents via email notifications.

Comment (Anurag810) [reply erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md]: done

File: erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md (hunk 2 above)
Comment (Anurag810) [reply erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md]: done

File: erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md (hunk 1 above)
Comment (Anurag810) [reply erpnext_com/www/docs/user/manual/en/setting-up/email/document-follow.md]: done

**Pull request merged**